packages/teams-js/src/private/store.ts

```diff
@@ -39,14 +40,25 @@ export enum StoreDialogType {
   AppDetail = 'appdetail',
 }
 
+/**
+ * @beta
+ * @hidden
+ * Interface of store dialog size
+ * @internal
+ * Limited to Microsoft-internal use
+ */
+export interface StoreSizeInfo {
+  size?: DialogSize;
```

Comment:
I actually don't have a problem with you re-using DialogSize here, but you should be aware that that object lets devs either specify specific numbers for the size *or* a more abstract small/medium/large. If you want to support the more abstract sizes that's fine, just be aware that it means the host will need to decide how big to make the dialogs in those cases.